AllReadyApp/Web-App/AllReady.UnitTest/Controllers/TaskApiControllerTests.cs

```diff
@@ -21,7 +21,7 @@
 
 namespace AllReady.UnitTest.Controllers
 {
-    public class TaskApiControllerTests
+    public class TaskApiControllerTests : InMemoryContextTest
```

Comment:
we should not inherit from InMemoryContextTest for controller tests. The primary purpose of InMemoryContextTest is to provide an in-memory EF context. Since none of our controllers use the AllReadyContext directly, and instead, use MediatR to send messages to "get stuff done", please remove this. When you do, four tests will break. check the rest of the review out for the fixes for those controllers